docs/tutorials/up12_initial_configuration.rst

```diff
@@ -12,9 +12,22 @@ Setting up the |motor_name| for the |propeller_name|
 
 For the |propeller_name| to create control torque, the |motor_name| must have the appropriate firmware and settings applied. You can download the pulsing firmware for the |motor_name| on the `underactuated propeller page <https://www.vertiq.co/up12>`_. To flash the downloaded firmware use the instructions in :ref:`updating_firmware`.
 
-Once the firmware is flashed, connect the module to IQ Control Center and go to the ‘General’ tab. Download the :download:`UP12 Custom Defaults File<../_static/defaults_files/custom_defaults_pulsing_12x4.json>`. Use the ‘Import’ button to load the defaults file into IQ Control Center.
+Once the firmware is flashed, connect the module to IQ Control Center and go to the ‘General’ tab.
 
-Use the ‘Set’ button to save the settings from the defaults file onto the |motor_name|. This will set all the required settings as well as change the |motor_name|’s baud rate. Reboot the |motor_name|. Once rebooted, to reconnect to IQ Control Center you must set the baud rate to 921600.
+.. warning::
+    With firmwares older than V0.0.27 defaults for the UP12 were not included in the firmware and custom defaults needed to be flashed. To do this download the :download:`UP12 Custom Defaults File<../_static/defaults_files/custom_defaults_pulsing_12x4.json>`. Use the ‘Import’ button to load the defaults file into IQ Control Center.
+    Use the ‘Set’ button to save the settings from the defaults file onto the |motor_name|. This will set all the required settings as well as change the |motor_name|’s baud rate. Reboot the |motor_name|. Once rebooted, to reconnect to IQ Control Center you must set the baud rate to 921600.
+
+We also recommend using a baudrate of 921600 to increase the throughput of the serial line. This can be done by going to the advanced tab in IQ Control Center and changing the baudrate setting.
```

Comment:
May want to capitalize "advanced tab" here

Reply:
At least the advanced part, advanced -> Advanced

Reply:
I capitalized and put quotes around it to match other parts of the page.